api/db/migrate/20170406200143_add_default_role_to_users.rb

```diff
@@ -1,6 +1,6 @@
 class AddDefaultRoleToUsers < ActiveRecord::Migration[5.0]
   def up
-    change_column :users, :role, :string, default: User::ROLE_READER
+    change_column :users, :role, :string, default: Role::ROLE_READER
     execute "UPDATE users SET role = 'reader' WHERE role IS NULL"
```

Comment:
I'm confused by this as well. With the introduction of Rollify, aren't user roles stored in the user_roles join table? If so, why do we also have a string field on the user model? This seems wrong—but if I'm misunderstanding, please explain.